### ISSUE TITLE
Navigation with Backgrid.Extension.Paginator

### ISSUE THREAD

**Issue by ThanksForAllTheFish** [wyuenho/backgrid#323]
Following the guide http://wyuenho.github.io/backgrid/#api-paginator I created a paginator like

``` javascript
this.fdtTable = new Backgrid.Grid({
    columns: this.columns,
    collection: this.collection
});

this.$el.empty();
this.$el.append(this.fdtTable.render().$el);

var paginator = new Backgrid.Extension.Paginator({
    collection: this.collection
});

this.$el.append(paginator.render().$el);
```

which initially renders as expected (so with a table and a paginator at the bottom of the table). However, when I try to navigate (both with the `fastForwardHandleLabels` or with the page number) I am not able to activate the labels to move back nor the page labeled `1`. I changed the `changePage` method to

``` javascript
changePage: function (e) {
      e.preventDefault();

      var $li = $(e.target).parent();
      if (!$li.hasClass("active") && !$li.hasClass("disabled")) {

        var label = $(e.target).text();
        var ffLabels = this.fastForwardHandleLabels;

        var collection = this.collection;

        if (ffLabels) {
          switch (label) {
          case ffLabels.first:
            collection.getFirstPage();
            this.render(); //Added by me
            return;
          case ffLabels.prev:
            collection.getPreviousPage();
            this.render(); //Added by me
            return;
          case ffLabels.next:
            collection.getNextPage();
            this.render(); //Added by me
            return;
          case ffLabels.last:
            collection.getLastPage();
            this.render(); //Added by me
            return;
          }
        }

        var state = collection.state;
        var pageIndex = +label;
        collection.getPage(state.firstPage === 0 ? pageIndex - 1 : pageIndex);
        this.render(); //Added by me
      }
    }
```

so basically forcing the rendering of the paginator after each interaction and now it works as expected. Did I misuconfigured the paginator in the first place?
